CONTRIBUTING.md

```diff
@@ -0,0 +1,255 @@
+- [Introduction](#introduction)
+- [The contribution process](#the-contribution-process)
+  - [Raise an issue](#raise-an-issue)
+  - [Create a fork](#create-a-fork)
+  - [Create a new folder](#create-a-new-folder)
+  - [Add license](#add-license)
+  - [Create a notebook](#create-a-notebook)
+  - [Commit new changes](#commit-new-changes)
+  - [Open a pull request](#open-a-pull-request)
+  - [Common recommendations for the review](#common-recommendations-for-the-review)
+- [CI/CD test passing guide](#cicd-test-passing-guide)
+  - [PEP 8 Style](#pep-8-style)
+  - [Notebook execution](#notebook-execution)
+  - [Format requirements](#format-requirements)
+- [Benchmarking result report](#benchmarking-result-report)
+
+## Introduction
+
+Thanks for considering a contribution to the MONAI Tutorials Repository and reading the Contributing Guidelines.
+
+The following is a set of guidelines for contributing tutorials to Project MONAI.
+The guidelines are based on the discussions in #1119
```

Comment:
I think no need to say the doc is based on some discussion ticket.
You can refer to: https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#communicate-with-us.

Thanks.

Reply:
Made some word changes of this line. Now it is:

If you have any questions about the guideline, please communicate with us and we are happy to discuss with you.
MONAI is an open-source project and relies on the community to grow and shine.
We want to use the guideline to help the growth of the project, instead of restricting the depth and breadth of your contribution.